cli/lib/exec/spawn.js

```diff
@@ -251,8 +251,12 @@ module.exports = {
             // the stderr logs unless we've explicitly
             // enabled the electron debug logging
             if (!debugElectron.enabled) {
```

Comment:
Since there was a PR to ignore garbage coming from Electron #4644, I'm almost wondering if we can just get rid of lines 253-259 and `return true` here. Theoretically, all other stderr should either be from the plugin, or from a genuine error in Cypress.

Reply:
There's a problem on Linux, where we always log Electron's Chromium stderr by enabling , which isn't filtered out:

https://github.com/cypress-io/cypress/blob/719fd98b15bf1382d795c86cb45030fa417053c9/cli/lib/exec/spawn.js#L247-L256

https://github.com/cypress-io/cypress/blob/719fd98b15bf1382d795c86cb45030fa417053c9/cli/lib/exec/spawn.js#L158-L160

So if a regex could be added to `GARBAGE_WARNINGS` to filter Electron's Chromium stderr, we can just remove lines 253-259 in `spawn.js`. 

We still need to keep these Chromium stderrs though, because we rely on them to know why Cypress crashed in some situations:

https://github.com/cypress-io/cypress/blob/719fd98b15bf1382d795c86cb45030fa417053c9/cli/lib/exec/spawn.js#L260-L262

Reply:
@santoshyadav198613 Do you want to try to implement this, or would you prefer me to do it?

Reply:
### 

> I am traveling till 8th Feb, Incan take it after that. But if you think it's urgent, please go ahead.